docs/contributing.md

```diff
@@ -47,11 +47,10 @@ Examples of bad PR title:
 
 ## Changes to `third_party`
 
-[`deno_third_party`](https://github.com/denoland/deno_third_party) contains most
-of the external code that Deno depends on, so that we know exactly what we are
-executing at any given time. It is carefully maintained with a mixture of manual
-labor and private scripts. It's likely you will need help from @ry or
-@piscisaureus to make changes.
+[`deno_third_party`](https://github.com/denoland/deno_third_party) contains
+prebuilt binaries used by deno that, if they were directly included, would take
+too long to build in the CI. It's likely you will need help from @ry or
+@piscisaureus to make changes there.
```

Comment:
We should remove this section altogether, there are only a couple tools in `third_party` and I don't expect they will need to be changed in foreseeable future

Reply:
Removed 👍